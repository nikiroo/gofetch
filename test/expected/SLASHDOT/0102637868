    ROBOT BOAT SAILS INTO HISTORY BY FINISHING ATLANTIC CROSSING   
                            (APNEWS.COM)                           

  Thursday September 06, 2018 @11:30PM (msmash)
  from the impressive-feat dept.

  o Reference: 0102637868
  o News link: https://hardware.slashdot.org/story/18/09/06/1719243/robot-boat-sails-into-history-by-finishing-atlantic-crossing
  o Source link: https://www.apnews.com/f6d0e2a099684468873ab48966590ada


    An  anonymous  reader  writes: For the first time an autonomous
    sailing  robot  has  completed  the  Microtransat  Challenge by
    crossing  the  Atlantic  from  Newfoundland, Canada to Ireland.
    The  Microtransat  has  been running since 2010 and has seen 23
    previous  entries  all  fail  to make it across. The successful
    boat,  SB  Met  was  built  by  the  Norwegian company Offshore
    Sensing  AS  and  is  only 2 metres (6.5 ft) long. It completed
    the  crossing  on August 26th, 79 days and 5000 km (3100 miles)
    of  sailing  after  departing Newfoundland on June 7th. Further
    reading:  A  Fleet  of  Sailing Robots Sets Out To Quantify the
    Oceans.


  ** 

     ** Re:Hard to understand why this would be difficult (Score:4,
        Informative)
        (by arth1 ( 260657 ))

        
        You're a landlubber, I take it?
        The Atlantic waves are quite an obstacle for a 2m boat.
        Even drifting tar (and garbage) is a problem when you're that
        small.


     ** Re: (Score:3, Funny)
        (by Anonymous Coward)

        
        It is not that easy. If you just start sailing in a straight
        line, you will end up many kilometers above the sea, which is
        no good for a sail boat. If instead you point the boat
        straight at the finish line, you'll be under water for most
        of the trip. Again, not ideal for a sail boat. The earth is
        not flat, you know.

        ** Re:Hard to understand why this would be difficult
           (Score:4, Funny)
           (by Oswald McWeany ( 2428506 ))

           
           I've found a FLAT EARTH DENIER !
           I bet you vaccinate your kids and don't believe in lizard
           people either.

           ** Re: (Score:2)
              (by gnick ( 1211984 ))

              
              Flat Earth statistic: If gravity is caused by a flat
              earth accelerating at g, it'll reach c after about 1
              year (~354 days).

              ** Re: (Score:2)
                 (by amorsen ( 7485 ))

                 
                 > Flat Earth statistic: If gravity is caused by a flat
                 > earth accelerating at g, it'll reach c after about 1
                 > year (~354 days).
                 You are saying that as if it is a problem to hit C,
                 like it would be impossible to continue after 354
                 days. This is not the case. As long as you only care
                 about your own point of view and you're the one
                 being accelerated, you can reach as many times C as
                 you want (well, have fuel for).

                 ** Re: (Score:2)
                    (by gnick ( 1211984 ))

                    
                    You also have to decide what "at rest" means. A
                    year accelerating at g will put you at your
                    current velocity + c.

                    ** 

                       ** Re: (Score:2)
                          (by lokedhs ( 672255 ))

                          
                          Would a flat earther believe in relativity
                          though? If simple geometry eludes them,
                          what would they do when being faced with a
                          Lorentz transformation?






           ** Re: (Score:2)
              (by lgw ( 121541 ))

              
              > I've found a FLAT EARTH DENIER!
              > I bet you vaccinate your kids and don't believe in
              > lizard people either.
              The earth is not flat - that's just silly. When we ent
              to the moon, and we did go to the moon, we discovered
              the terrifying truth. The MOON is flat!




     ** Re:Hard to understand why this would be difficult (Score:4,
        Insightful)
        (by hey! ( 33014 ))

        
        > There really aren't a lot of obstacles to deal with. Just
        > point the boat and go.
        I imagine that's true, if you have an effectively unlimited
        energy, but the terms of this contest is to build a vehicle
        less than eight feet long that can cross the Atlantic.
        That's not a lot of room to stuff with batteries or diesel
        fuel, or to cover with solar panels. And that's the whole
        point. If you could make the vehicle a hundred feet long,
        this challenge would be expensive, but easy. By making the
        boat tiny, you make the challenge affordable, but tough.
        The race has two divisions, sail powered and unlimited, but
        it's hard to see what they had in mind for propulsion by
        having an unlimited division. Possibly some
        seawater-replenished fuel cell.

        ** 

           ** Bottles with corks have traversed the Atlantic (Score:2)
              (by aberglas ( 991072 ))

              
              Being small means relative high strength. A solid two
              meter boat should indestructible by waves. No need to
              keep a human alive, nor to go very fast. A very basic,
              fixed, small but strong sail would do.
              Ice bergs could be a problem. Surely better to start
              further south. Longer but safer. Also need to stay out
              of shipping lanes.
              I think this could be done without a computer. Just a
              magnetic compass controlling a rudder. Occasionally it
              would be blown backwards but no big deal. No need to
              worry about po




  ** KABLAM (Score:1)
     (by fluffernutter ( 1411889 ))

     
     No concrete barriers in the ocean?

     ** Re: (Score:2)
        (by zlives ( 2009072 ))

        
        or pedestrians


  ** A small achievement... (Score:2)
     (by Bearhouse ( 1034238 ))

     
     Sure the Atlantic can be a tough place, but 79 days?
     Single-handed (human) record is 3 1/2 days...
     [1]https://en.wikipedia.org/wiki/... [wikipedia.org]
     
     
     
     
     [1] https://en.wikipedia.org/wiki/Transatlantic_sailing_record

     ** Re:A small achievement... (Score:4, Informative)
        (by careysub ( 976506 ))

        
        This is a 2 meter vessel, not a large racing hull built to be
        fast (and nothing but fast). Using the page you link to, the
        most useful comparison would be the single-handed records,
        and the one from 1987 which was about 11.5 days was in a 26
        meter hull! And this is the smallest vessel on the list. You
        are probably not going to get a 2 meter vessel to tear along
        at an average speed of 7.5 m/sec which would be needed for
        that 11.5 day crossing.

        ** Re: (Score:3)
           (by Solandri ( 704621 ))

           
           There have been a few sailboats in the 2 meter size range
           which have made the transatlantic crossing.
           
           Hugo Vihlen aboard April Fool (1.8m) in 84 days in 1968,
           though technically he never finished since he was picked
           up by the USCG 6 miles offshore.
           Tom MacNally aboard the Vera Hugh (1.64m) in 134 days in
           1993.
           Hugo Vihlen again aboard the Father's Day (1.62m) in 115
           days in 1993.
           Tom MacNally unsuccessfully attempted the trip again
           aboard a 1.19m craft in 1998. He passed away last year.
           [1]Site with picture [microcruising.com]
           
           
           
           
           [1] http://www.microcruising.com/famoussmallboats.htm



     ** Re: (Score:3)
        (by careysub ( 976506 ))

        
        Also note that the world speed record for a radio-controlled
        sailing vessel (which was also 2 meters long) is [1]157.65 km
        in 23 hours 42 min [guinnessworldrecords.com] in the milder
        waters of the Mediterranean. At that world record speed in a
        small RC boat this challenge would have taken 32 days.
        So 79 days in a boat not controlled by a human, and not
        limited to a 24 hour period, in the open waters of the
        Atlantic is not so shabby. The speed made good over that
        whole journey is 40% of that 24 hour record.
        
        
        
        
        [1]
        http://www.guinnessworldrecords.com/world-records/greatest-di-
        stance-by-radio-controlled-model-sailing-ship-in-24-hours-(rc)


     ** Re: (Score:3)
        (by arth1 ( 260657 ))

        
        > Sure the Atlantic can be a tough place, but 79 days?
        The Mayflower took 66 days to cross, and was much bigger.


  ** 

     ** Re: (Score:2)
        (by zlives ( 2009072 ))

        
        shipping containers are more profitable.


  ** And now the jokes begin... (Score:1)
     (by Jharish ( 101858 ))

     
     ..."I rowboat".
     https://www.theonion.com/i-rowboat-1819583491

     ** Re: (Score:2)
        (by starless ( 60879 ))

        
        Followed later by Cory Doctorow
        [1]http://bestsciencefictionstori... [bestscienc...tories.com]
        
        
        
        
        [1]
        http://bestsciencefictionstories.com/2009/03/29/i-row-boat-by-
        cory-doctorow/


  ** Description wrong? Not autonomous? (Score:5, Informative)
     (by starless ( 60879 ))

     
     > For the first time an autonomous sailing robot...
     From the linked article:
     [1]https://www.apnews.com/f6d0e2a... [apnews.com]
     > The Sailbuoy competed in the “unmanned” class, which allows
     > operators to change its course along the way. There’s a separate
     > “autonomous” class that prohibits any such communication.
     >> 
     
     
     
     
     [1] https://www.apnews.com/f6d0e2a099684468873ab48966590ada

  ** So you crossed the sea, Columbot (Score:1)
     (by DulcetTone ( 601692 ))

     
     Don't forget that indigenous robots have rights, too.


