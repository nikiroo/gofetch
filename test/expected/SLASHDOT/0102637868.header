0Robot Boat Sails Into History By Finishing Atlantic Crossing (apnews.com)	null/SLASHDOT/0102637868		70
i  Thursday September 06, 2018 @11:30PM (msmash)
i  from the impressive-feat dept.
i
i    An  anonymous  reader  writes: For the first time an autonomous
i    sailing  robot  has  completed  the  Microtransat  Challenge by
i    crossing  the  Atlantic  from  Newfoundland, Canada to Ireland.
i    The  Microtransat  has  been running since 2010 and has seen 23
i    previous  entries  all  fail  to make it across. The successful
i    boat,  SB  Met  was  built  by  the  Norwegian company Offshore
i    Sensing  AS  and  is  only 2 metres (6.5 ft) long. It completed
i    the  crossing  on August 26th, 79 days and 5000 km (3100 miles)
i    of  sailing  after  departing Newfoundland on June 7th. Further
i    reading:  A  Fleet  of  Sailing Robots Sets Out To Quantify the
i    Oceans.
i
