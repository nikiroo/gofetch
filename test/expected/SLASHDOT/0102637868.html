<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Robot Boat Sails Into History By Finishing Atlantic Crossing (apnews.com)</h2>
	<div class='details'>(Thursday September 06, 2018 @11:30PM (msmash) 
from the impressive-feat dept.)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0102637868</a></li>
        <li>News link: <a href='https://hardware.slashdot.org/story/18/09/06/1719243/robot-boat-sails-into-history-by-finishing-atlantic-crossing'>https://hardware.slashdot.org/story/18/09/06/1719243/robot-boat-sails-into-history-by-finishing-atlantic-crossing</a></li>
        <li>Source link: <a href='https://www.apnews.com/f6d0e2a099684468873ab48966590ada'>https://www.apnews.com/f6d0e2a099684468873ab48966590ada</a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		An anonymous reader writes:<br/><br/>&gt; For the first time an autonomous sailing robot has completed the [1]Microtransat Challenge by [2]crossing the Atlantic from Newfoundland, Canada to Ireland . The Microtransat has been running since 2010 and has seen 23 previous entries all fail to make it across. The successful boat, [3]SB Met was built by the Norwegian company [4]Offshore Sensing AS and is only 2 metres (6.5 ft) long. It completed the crossing on August 26th, 79 days and 5000 km (3100 miles) of sailing after departing Newfoundland on June 7th.<br/><br/>Further reading:  [5]A Fleet of Sailing Robots Sets Out To Quantify the Oceans .<br/><br/><br/><br/>[1] https://www.microtransat.org/<br/><br/>[2] https://www.apnews.com/f6d0e2a099684468873ab48966590ada<br/><br/>[3] https://www.microtransat.org/2018_sailbuoy_boat.php<br/><br/>[4] http://sailbuoy.no/<br/><br/>[5] https://science.slashdot.org/story/18/05/16/228245/a-fleet-of-sailing-robots-sets-out-to-quantify-the-oceans
	</div>
<hr/>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3></h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re:Hard to understand why this would be difficult (Score:4, Informative)</h3>
      <div class='by' style='font-style: italic;'>by arth1 ( 260657 )</div>
      <div class='comment_content'><p></p><p>You're a landlubber, I take it?</p><p>The Atlantic waves are quite an obstacle for a 2m boat.</p><p>Even drifting tar (and garbage) is a problem when you're that small.</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:3, Funny)</h3>
      <div class='by' style='font-style: italic;'>by Anonymous Coward</div>
      <div class='comment_content'><p></p><p>It is not that easy. If you just start sailing in a straight line, you will end up many kilometers above the sea, which is no good for a sail boat. If instead you point the boat straight at the finish line, you'll be under water for most of the trip. Again, not ideal for a sail boat. The earth is not flat, you know.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3>Re:Hard to understand why this would be difficult (Score:4, Funny)</h3>
        <div class='by' style='font-style: italic;'>by Oswald McWeany ( 2428506 )</div>
        <div class='comment_content'><p></p><p>I've found a FLAT EARTH DENIER !</p><p>I bet you vaccinate your kids and don't believe in lizard people either.</p></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h3>Re: (Score:2)</h3>
          <div class='by' style='font-style: italic;'>by gnick ( 1211984 )</div>
          <div class='comment_content'><p></p><p>Flat Earth statistic: If gravity is caused by a flat earth accelerating at g, it'll reach c after about 1 year (~354 days).</p></div>
          <div class='comment' style='display: block; margin-left: 80px'>
            <h3>Re: (Score:2)</h3>
            <div class='by' style='font-style: italic;'>by amorsen ( 7485 )</div>
            <div class='comment_content'><p></p><p>> Flat Earth statistic: If gravity is caused by a flat earth accelerating at g, it'll reach c after about 1 year (~354 days).</p><p>You are saying that as if it is a problem to hit C, like it would be impossible to continue after 354 days. This is not the case. As long as you only care about your own point of view and you're the one being accelerated, you can reach as many times C as you want (well, have fuel for).</p></div>
            <div class='comment' style='display: block; margin-left: 80px'>
              <h3>Re: (Score:2)</h3>
              <div class='by' style='font-style: italic;'>by gnick ( 1211984 )</div>
              <div class='comment_content'><p></p><p>You also have to decide what "at rest" means. A year accelerating at g will put you at your current velocity + c.</p></div>
              <div class='comment' style='display: block; margin-left: 80px'>
                <h3></h3>
                <div class='by' style='font-style: italic;'></div>
                <div class='comment_content'></div>
                <div class='comment' style='display: block; margin-left: 80px'>
                  <h3>Re: (Score:2)</h3>
                  <div class='by' style='font-style: italic;'>by lokedhs ( 672255 )</div>
                  <div class='comment_content'><p></p><p>Would a flat earther believe in relativity though? If simple geometry eludes them, what would they do when being faced with a Lorentz transformation?</p></div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h3>Re: (Score:2)</h3>
          <div class='by' style='font-style: italic;'>by lgw ( 121541 )</div>
          <div class='comment_content'><p></p><p>> I've found a FLAT EARTH DENIER!</p><p>> I bet you vaccinate your kids and don't believe in lizard people either.</p><p>The earth is not flat - that's just silly. When we ent to the moon, and we did go to the moon, we discovered the terrifying truth. The MOON is flat!</p></div>
        </div>
      </div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re:Hard to understand why this would be difficult (Score:4, Insightful)</h3>
      <div class='by' style='font-style: italic;'>by hey! ( 33014 )</div>
      <div class='comment_content'><p></p><p>> There really aren't a lot of obstacles to deal with. Just point the boat and go.</p><p>I imagine that's true, if you have an effectively unlimited energy, but the terms of this contest is to build a vehicle less than eight feet long that can cross the Atlantic.</p><p>That's not a lot of room to stuff with batteries or diesel fuel, or to cover with solar panels. And that's the whole point. If you could make the vehicle a hundred feet long, this challenge would be expensive, but easy. By making the boat tiny, you make the challenge affordable, but tough.</p><p>The race has two divisions, sail powered and unlimited, but it's hard to see what they had in mind for propulsion by having an unlimited division. Possibly some seawater-replenished fuel cell.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3></h3>
        <div class='by' style='font-style: italic;'></div>
        <div class='comment_content'></div>
        <div class='comment' style='display: block; margin-left: 80px'>
          <h3>Bottles with corks have traversed the Atlantic (Score:2)</h3>
          <div class='by' style='font-style: italic;'>by aberglas ( 991072 )</div>
          <div class='comment_content'><p></p><p>Being small means relative high strength. A solid two meter boat should indestructible by waves. No need to keep a human alive, nor to go very fast. A very basic, fixed, small but strong sail would do.</p><p>Ice bergs could be a problem. Surely better to start further south. Longer but safer. Also need to stay out of shipping lanes.</p><p>I think this could be done without a computer. Just a magnetic compass controlling a rudder. Occasionally it would be blown backwards but no big deal. No need to worry about po</p></div>
        </div>
      </div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>KABLAM (Score:1)</h3>
    <div class='by' style='font-style: italic;'>by fluffernutter ( 1411889 )</div>
    <div class='comment_content'><p></p><p>No concrete barriers in the ocean?</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:2)</h3>
      <div class='by' style='font-style: italic;'>by zlives ( 2009072 )</div>
      <div class='comment_content'><p></p><p>or pedestrians</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>A small achievement... (Score:2)</h3>
    <div class='by' style='font-style: italic;'>by Bearhouse ( 1034238 )</div>
    <div class='comment_content'><p></p><p>Sure the Atlantic can be a tough place, but 79 days?</p><p>Single-handed (human) record is 3 1/2 days...</p><p>[1]https://en.wikipedia.org/wiki/... [wikipedia.org]</p><p></p><p></p><p></p><p></p><p>[1] https://en.wikipedia.org/wiki/Transatlantic_sailing_record</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re:A small achievement... (Score:4, Informative)</h3>
      <div class='by' style='font-style: italic;'>by careysub ( 976506 )</div>
      <div class='comment_content'><p></p><p>This is a 2 meter vessel, not a large racing hull built to be fast (and nothing but fast). Using the page you link to, the most useful comparison would be the single-handed records, and the one from 1987 which was about 11.5 days was in a 26 meter hull! And this is the smallest vessel on the list. You are probably not going to get a 2 meter vessel to tear along at an average speed of 7.5 m/sec which would be needed for that 11.5 day crossing.</p></div>
      <div class='comment' style='display: block; margin-left: 80px'>
        <h3>Re: (Score:3)</h3>
        <div class='by' style='font-style: italic;'>by Solandri ( 704621 )</div>
        <div class='comment_content'><p></p><p>There have been a few sailboats in the 2 meter size range which have made the transatlantic crossing.</p><p></p><p>Hugo Vihlen aboard April Fool (1.8m) in 84 days in 1968, though technically he never finished since he was picked up by the USCG 6 miles offshore.</p><p>Tom MacNally aboard the Vera Hugh (1.64m) in 134 days in 1993.</p><p>Hugo Vihlen again aboard the Father's Day (1.62m) in 115 days in 1993.</p><p>Tom MacNally unsuccessfully attempted the trip again aboard a 1.19m craft in 1998. He passed away last year.</p><p>[1]Site with picture [microcruising.com]</p><p></p><p></p><p></p><p></p><p>[1] http://www.microcruising.com/famoussmallboats.htm</p></div>
      </div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:3)</h3>
      <div class='by' style='font-style: italic;'>by careysub ( 976506 )</div>
      <div class='comment_content'><p></p><p>Also note that the world speed record for a radio-controlled sailing vessel (which was also 2 meters long) is [1]157.65 km in 23 hours 42 min [guinnessworldrecords.com] in the milder waters of the Mediterranean. At that world record speed in a small RC boat this challenge would have taken 32 days.</p><p>So 79 days in a boat not controlled by a human, and not limited to a 24 hour period, in the open waters of the Atlantic is not so shabby. The speed made good over that whole journey is 40% of that 24 hour record.</p><p></p><p></p><p></p><p></p><p>[1] http://www.guinnessworldrecords.com/world-records/greatest-distance-by-radio-controlled-model-sailing-ship-in-24-hours-(rc)</p></div>
    </div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:3)</h3>
      <div class='by' style='font-style: italic;'>by arth1 ( 260657 )</div>
      <div class='comment_content'><p></p><p>> Sure the Atlantic can be a tough place, but 79 days?</p><p>The Mayflower took 66 days to cross, and was much bigger.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3></h3>
    <div class='by' style='font-style: italic;'></div>
    <div class='comment_content'></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:2)</h3>
      <div class='by' style='font-style: italic;'>by zlives ( 2009072 )</div>
      <div class='comment_content'><p></p><p>shipping containers are more profitable.</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>And now the jokes begin... (Score:1)</h3>
    <div class='by' style='font-style: italic;'>by Jharish ( 101858 )</div>
    <div class='comment_content'><p></p><p>..."I rowboat".</p><p>https://www.theonion.com/i-rowboat-1819583491</p></div>
    <div class='comment' style='display: block; margin-left: 80px'>
      <h3>Re: (Score:2)</h3>
      <div class='by' style='font-style: italic;'>by starless ( 60879 )</div>
      <div class='comment_content'><p></p><p>Followed later by Cory Doctorow</p><p>[1]http://bestsciencefictionstori... [bestscienc...tories.com]</p><p></p><p></p><p></p><p></p><p>[1] http://bestsciencefictionstories.com/2009/03/29/i-row-boat-by-cory-doctorow/</p></div>
    </div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>Description wrong? Not autonomous? (Score:5, Informative)</h3>
    <div class='by' style='font-style: italic;'>by starless ( 60879 )</div>
    <div class='comment_content'><p></p><p>> For the first time an autonomous sailing robot...</p><p>From the linked article:</p><p>[1]https://www.apnews.com/f6d0e2a... [apnews.com]</p><p>> The Sailbuoy competed in the “unmanned” class, which allows operators to change its course along the way. There’s a separate “autonomous” class that prohibits any such communication.</p><p>>></p><p></p><p></p><p></p><p></p><p>[1] https://www.apnews.com/f6d0e2a099684468873ab48966590ada</p></div>
  </div>
  <div class='comment' style='display: block; margin-left: 80px'>
    <h3>So you crossed the sea, Columbot (Score:1)</h3>
    <div class='by' style='font-style: italic;'>by DulcetTone ( 601692 )</div>
    <div class='comment_content'><p></p><p>Don't forget that indigenous robots have rights, too.</p></div>
  </div>
</div>
</body>
